node/plan.ps1

```diff
@@ -1,7 +1,7 @@
 $pkg_name="node"
 $pkg_origin="core"
 $pkg_version="12.13.1"
-$pkg_description="Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine."
+$pkg_description="Node.jsÃ‚Â® is a JavaScript runtime built on Chrome's V8 JavaScript engine."
```

Comment:
nit: are those weird characters expected? 🤔

Reply:
good catch! No they are not but I know why this happened. There is a bug in the PSScriptAnalyzer's `-Fix` arg that prepends a BOM to any file it touches. So I ran all these files though a script that stipped the BOM. I noticed a couple plans afterwards that had characters like this inserted before special characters like this. So my guess there was an encoding issue in that filter. I will fix this.